http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/common/Constants.java

```diff
@@ -129,7 +129,7 @@
 
     public static final String PROTOCOL = "PROTOCOL";
 
-    public static final String PROTOCOL_NAME = "http";
```

Comment:
Can we deprecate this for this PR and remove it later? Because, it will be backward incompatible release if we directly remove a public constant, which is probably used  in other repos.